### ISSUE TITLE
route53 resolver endpoints do not get created when networkFirewall is undefined

### ISSUE THREAD

**Issue by steerben** [awslabs/landing-zone-accelerator-on-aws#152]
**Describe the bug**
Within the `network-vpc-endpoints-stack` the creation of the route53 resolver resources (_specifically within that stack the Route53 resolver endpoints_) ONLY get created when `centralNetworkServicesConfig.networkFirewall.firewalls` property is set in the `network-config.yaml` in the config repo.

Reason for this is the `if` statement https://github.com/awslabs/landing-zone-accelerator-on-aws/blob/v1.4.0/source/packages/%40aws-accelerator/accelerator/lib/stacks/network-stacks/network-vpc-endpoints-stack/network-vpc-endpoints-stack.ts#L89 which also contains the **independent** creation of the route53 resolver endpoints further down.

Problematic here is esp. that the LZA pipeline fails if you additionally configure **resolver rules**. 
Those reference the endpoint IDs in SSM Param Store which never get set and therefore create a pipeline failure

**To Reproduce**
```
centralNetworkServices:
  delegatedAdminAccount: SharedServices
  route53Resolver:
    endpoints:
      - name: central-dns-resolver-outbound
        type: OUTBOUND
        vpc: network-services
        allowedCidrs:
          - 192.168.140.1/32
          - 192.168.140.111/32
        subnets:
          - network-services-a
          - network-services-b
      - name: central-dns-resolver-inbound
        type: INBOUND
        vpc: network-services
        allowedCidrs:
          - 192.168.140.1/32
          - 192.168.140.111/32
        subnets:
          - network-services-a
          - network-services-b
```

Missing here is the following block, which enables the expected behaviour.

```
  networkFirewall:
    firewalls: []
    policies: []
    rules: []
```

**Expected behavior**
Creation of route53 resolver endpoints without defining `networkFirewall` object in the `network-config.yaml`.
Route53 resolver endpoint creation is independent of networkFirewall configuration/resources.

**Please complete the following information about the solution:**

- [ ] Version: v1.4.0
- [ ] Region: eu-central-1
- [ ] Was the solution modified from the version published on this repository? - No
- [ ] If the answer to the previous question was yes, are the changes available on GitHub? - No
- [ ] Have you checked your [service quotas](https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html) for the sevices this solution uses? - Not applicable
- [ ] Were there any errors in the CloudWatch Logs? - See problem statement

**Screenshots**

**Additional context**

**Comment by awsclemj** [comment]
Hello @steerben, and thank you for reporting this! We are aware of the issue and are planning a patch release to fix this along with other issues in 1.4.0. We will keep you updated when that new release has been published.

**Comment by awsclemj** [comment]
Hello, this issue has been fixed in [v1.4.1](https://github.com/awslabs/landing-zone-accelerator-on-aws/releases/tag/v1.4.1). Thanks again for your bug report! I will go ahead and mark this issue as completed.
